src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -205,6 +205,7 @@ internal JObject LoadHostConfig(string configFilePath)
             private JObject GetDefaultHostConfigObject()
             {
                 var hostJsonJObj = JObject.Parse("{'version': '2.0'}");
+                hostJsonJObj.Add("extensionBundle", JToken.Parse("{ 'id': 'Microsoft.Azure.Functions.ExtensionBundle', 'version': '[1.*, 2.0.0)'}"));
```

Comment:
You don't need to do this here (we can follow-up) -- we should just be loading the host.json from resources here rather than manually creating it.

Reply:
Since this is an existing process, at the very least, to it all in the same `JObject.Parse` call